source/agora/node/Registry.d

```diff
@@ -914,6 +917,9 @@ private struct ZoneData
         this.query_records_get_typed = format("SELECT address, type, ttl " ~
             "FROM registry_%s_addresses WHERE pubkey = ? AND type = ?", zone_name);
 
+        this.query_records_get_all_typed = format("SELECT pubkey, address, type, ttl " ~
+            "FROM registry_%s_addresses WHERE type = ?", zone_name);
+
         this.query_utxo_get = format("SELECT sequence, utxo " ~
             "FROM registry_%s_utxo WHERE pubkey = ?", zone_name);
```

Comment:
Also here the number of params is correct but wrong variable I think.

Reply:
As first comment

Reply:
🤦‍♂️ 🤦‍♂️